VideoLocker/src/main/java/org/edx/mobile/model/api/TranscriptModel.java

```diff
@@ -69,4 +69,25 @@
         String label = context.getResources().getString(resId);
         return label;
     }*/
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
```

Comment:
Again, this is auto-generated code by Android Studio for equals method, I don't feel like modifying it.